core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -30,18 +40,26 @@
 
     public static final String NAME = "missing";
 
-    private String name;
+    public static final boolean DEFAULT_NULL_VALUE = false;
 
-    private Boolean nullValue;
+    public static final boolean DEFAULT_EXISTENCE_VALUE = true;
 
-    private Boolean existence;
+    private String fieldPattern;
+
+    private Boolean nullValue = DEFAULT_NULL_VALUE;
+
+    private Boolean existence = DEFAULT_EXISTENCE_VALUE;
 
     static final MissingQueryBuilder PROTOTYPE = new MissingQueryBuilder(null);
 
-    public MissingQueryBuilder(String name) {
-        this.name = name;
+    public MissingQueryBuilder(String fieldPattern) {
+        this.fieldPattern = fieldPattern;
```

Comment:
I don't see this comment addressed yet..... it is subtle but checking for null in constructor is a bit different compared to validate, as I explained in my previous comment. Please let me know if  that is not clear yet, otherwise please address this comment.